app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= render 'shared/partials/nav-global.html' %>
```

Comment:
Right now we have a default layout for all our pages as a wrapper with the stylesheet and javascript file. I asked Jason about creating a layout for these separate pages.